include/worker.hpp

```diff
@@ -100,6 +101,11 @@ class Worker
                       types::ObjectMap& objectInterfaceMap,
                       const std::string& vpdFilePath);
 
+    /**
+     * @brief An API to backup and restore VPD.
+     */
+    std::tuple<types::VPDMapVariant, types::VPDMapVariant> backupAndRestore();
```

Comment:
Don't see a need of this public API.

Reply:
Removed this API, this PR is only including BackupRestore API's implementation.